release-please-config.json

```diff
@@ -17,7 +17,8 @@
         "extra-files": [
```

Comment:
Maybe we just exclude the helm charts for updates to gitops serer.

Reply:
let's try it first, if it doesn't work, then sure.